hztb-rest-api/src/main/java/com/svs/hztb/api/sm/model/opinion/RequestOpinionInput.java

```diff
@@ -12,13 +11,14 @@
 	private int channelId;
 	private String storeGeoCode;
 	private String productUrl;
-	//Added for sample, delete this
+	// Added for sample, delete this
 	private Date date;
-	
-	//private int requestedGroupId;
+
+	// private int requestedGroupId;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)